fleet_vehicle_inspection/models/fleet_vehicle_inspection_line.py

```diff
@@ -20,29 +20,39 @@ class FleetVehicleInspectionLine(models.Model):
         string='Inspection Reference',
         required=True,
         ondelete='cascade',
-        index=True,
-        copy=False)
+        index=True)
 
     inspection_item_id = fields.Many2one(
         'fleet.vehicle.inspection.item',
         'Inspection Item',
         required=True,
         track_visibility="onchange",
-        help='Inspection Item',
+        index=True,
+        ondelete='cascade',
         states=READONLY_STATES,
+        copy=True
     )
 
+    inspection_item_instruction = fields.Text(
+        'Instructions',
+        related='inspection_item_id.instruction',
+        size=140)
```

Comment:
Why `size=140` ?
Specially it being a related field..

Reply:
I thought about the size of the field per house of the kanban mobile but it is better not to limit it.